docs/t-sql/statements/drop-database-transact-sql.md

```diff
@@ -161,9 +158,27 @@ The following example removes a database snapshot, named `sales_snapshot0600`, w
 DROP DATABASE sales_snapshot0600;
 ```
 
+### D. Dropping a database if it exists
+
+The following example removes a database named `Sales` if it exists, by first changing to single user mode.
+
+```sql
+USE tempdb;
+GO
+DECLARE @SQL nvarchar(1000);
+IF EXISTS (SELECT 1 FROM sys.databases WHERE [name] = N'Sales')
+BEGIN
+    SET @SQL = N'USE tempdb;
+                 ALTER DATABASE Sales SET SINGLE_USER WITH ROLLBACK IMMEDIATE;
+                 USE master;

```

Comment:
```suggestion
                 USE [tempdb];

```

Reply:
Hi William, 

That line 171 needs to be USE Sales; not USE master;